src/Driver/driver_configs.jl

```diff
@@ -404,6 +406,60 @@ Establishing Atmos GCM configuration for %s
     )
 end
 
+export EmptyParamSet
+export init_empty!, get_grid_spacing_les, get_grid_spacing_gcm
+struct EmptyParamSet <: AbstractEarthParameterSet end
+function init_empty!(problem, bl, state, aux, localgeo, t)
+    nothing
+end
+function get_grid_spacing_les(
+    N_poly::Union{Int, NTuple{2, Int}},
+    (Δx, Δy, Δz)::NTuple{3, FT},
+    xmax::FT,
+    ymax::FT,
+    zmax::FT;
+    init_empty::Function = init_empty!,
+) where {FT}
+    empty_config = ClimateMachine.AtmosLESConfiguration(
+        "EmptyModel",
+        N_poly,
+        (Δx, Δy, Δz),
+        xmax,
+        ymax,
+        zmax,
+        EmptyParamSet(),
+        init_empty;
+        suppress_model_info = true,
+    )
+    Δh = min_node_distance(empty_config.grid, HorizontalDirection())
+    Δv = min_node_distance(empty_config.grid, VerticalDirection())
+    return (Δh, Δv)
+end
+
+function get_grid_spacing_gcm(
+    N_poly::Union{Int, NTuple{2, Int}},
+    (nelem_horz, nelem_vert)::NTuple{2, Int},
+    domain_height::FT,
+    init_empty::Function = init_empty!,
+) where {FT}
+    empty_config = ClimateMachine.AtmosGCMConfiguration(
+        "EmptyModel",
+        N_poly,
+        (nelem_horz, nelem_vert),
+        domain_height,
+        EmptyParamSet(),
+        init_empty;
+        suppress_model_info = true,
+    )
+    Δh = min_node_distance(empty_config.grid, HorizontalDirection())
+    Δv = min_node_distance(empty_config.grid, VerticalDirection())
```

Comment:
I'm a bit confused. Is this calculating the grid size before declaring any discretization (i.e., either SEM or FV, or any other type)? If so, how does this differ from the [resolution](https://github.com/CliMA/ClimateMachine.jl/blob/6026b0a2ea86e6fdf0a3fc764635a4b8a81000ca/src/Driver/driver_configs.jl#L205) used to compute the [elemsize](https://github.com/CliMA/ClimateMachine.jl/blob/6026b0a2ea86e6fdf0a3fc764635a4b8a81000ca/src/Driver/driver_configs.jl#L225)?

Reply:
maybe along those lines - the name could be `get_min_node_distance` instead of `get_grid_spacing` to be clearer?

Reply:
Updated names for clarity

Reply:
> I'm a bit confused. Is this calculating the grid size before declaring any discretization (i.e., either SEM or FV, or any other type)? If so, how does this differ from the [resolution](https://github.com/CliMA/ClimateMachine.jl/blob/6026b0a2ea86e6fdf0a3fc764635a4b8a81000ca/src/Driver/driver_configs.jl#L205) used to compute the [elemsize](https://github.com/CliMA/ClimateMachine.jl/blob/6026b0a2ea86e6fdf0a3fc764635a4b8a81000ca/src/Driver/driver_configs.jl#L225)?

In the current driver setup it uses the empty model to get the discretization type; then allows this to be passed to `<: BalanceLaw`

Reply:
Thanks @akshaysridhar Shouldn't also the name of the return variables, e.g. `Δh` and `Δv`, should reflect this? Say, call them `minΔh`, `minΔv`? I am just a bit concerned because earlier on  in this same file, we use `(Δx, Δy, Δz) = ClimateMachine.Settings.resolution` in `AtmosLESConfiguration` and that is for the average node distance, not the min one